### PR TITLE
[ADMINAPI-970] Extend Docker Deployment for Admin API 2.1

### PR DESCRIPTION
**Pull request by CSR2017**
@simpat-jesus 
Notes: Admin API 2.1 supports multi-tenancy. May need to update the Admin API tag to 2.1 and ODS API to 7.1 for testing. Thanks

### CODE REVIEW

**Review by jagudelo-gap — COMMENTED** [review]
@CSR2017 
Please include Admin API in the README files

- Compose\pgsql\README.md
- Compose\pgsql\MultiTenant\README.md
- Compose\pgsql\MultiTenant-OdsContext\README.md

**Review by simpat-jesus — CHANGES_REQUESTED** [review]
Please modify the image for Admin Db containers for MultiTenant and MultiTenant with OdsContexts to be ${ADMIN_DB_IMAGE:-edfialliance/ods-api-db-admin:${TAG}}

Also, enable AllowRegistration and Swager; since this are just examples, users should be able to run the examples without modifying the compose files.

File: Compose-Generator/Alpine/templates/pgsql/MultiTenant-OdsContext/compose-template.mustache [hunk 1]
```diff
@@ -129,6 +131,41 @@ services:
       start_period: "60s"
       retries: 3
 
+  adminapi:
+      image: edfialliance/ods-admin-api:${ADMINAPI_TAG}
```
Comment (simpat-jesus) [inline Compose-Generator/Alpine/templates/pgsql/MultiTenant-OdsContext/compose-template.mustache:135]: Indentation has two levels and should have only 1 level. Please verify the changes here match the generated files using the Compose-Generator

File: Compose-Generator/Alpine/templates/pgsql/MultiTenant/compose-template.mustache [hunk 2]
```diff
@@ -49,6 +49,7 @@ services:
     image: edfialliance/ods-api-web-gateway:${GATEWAY_TAG}
     environment:
       ODS_VIRTUAL_NAME: "${ODS_VIRTUAL_NAME:-api}"
+      ADMIN_API_VIRTUAL_NAME: "${ADMIN_API_VIRTUAL_NAME:-adminapi}"
```
Comment (simpat-jesus) [inline Compose-Generator/Alpine/templates/pgsql/MultiTenant/compose-template.mustache:52]: There is a space at the end; this does not match the generated file. Please verify the changes here match the generated files using the Compose-Generator

File: Compose-Generator/Alpine/templates/pgsql/MultiTenant/compose-template.mustache [hunk 3]
```diff
@@ -96,6 +98,41 @@ services:
       start_period: "60s"
       retries: 3
 
+  adminapi:
+      image: edfialliance/ods-admin-api:${ADMINAPI_TAG}
```
Comment (simpat-jesus) [inline Compose-Generator/Alpine/templates/pgsql/MultiTenant/compose-template.mustache:102]: Indentation has two levels and should have only 1 level. Please verify the changes here match the generated files using the Compose-Generator

**Review by simpat-jesus — CHANGES_REQUESTED** [review]

**Review by jagudelo-gap — APPROVED** [review]

**Review by simpat-jesus — APPROVED** [review]

**Pull request merged**